coding/browser-support.md

```diff
@@ -4,20 +4,20 @@ layout: base
 
 # Browser support
 
-Our baseline browser support for our websites. There are two levels of support. 
+Our baseline browser support for our websites. There are two levels of support.
 
 ## Full support
 
 Any browser with full support needs to be pixel perfect and all functionality should work.
 
 | Browser | Version |
 | ------- | ------- |
-| Chrome | 55 or greater |
-| Firefox | 52 or greater |
-| Safari | 11 or greater |
+| Chrome | Should work for the last 3 [stable versions](https://www.chromestatus.com/features/schedule) |
+| Firefox | Should work for the last 3 [stable versions](https://wiki.mozilla.org/Release_Management/Calendar) |
+| Safari | [latest version](https://developer.apple.com/safari/technology-preview/release-notes) |
 | IE | 11 or greater |
-| Edge | 15 or greater |
-| Opera | 48 or greater |
+| Edge | Latest |
+| Opera | Should work for the last 3 [stable versions](https://www.opera.com/docs/history/#history) |
```

Comment:
Do we need last 3 for Opera if we only do latest for Edge or Safari? Was this decision based on stats? 'cos my understanding was that Opera tends to trail behind all the others in market share.

Reply:
Its based on the release cadence. Edge or Safari release much more slowly then Opera, etc.